README.md

```diff
@@ -37,3 +37,142 @@ Learn more about Turborepo:
+SES (email) or ACM (cert for CloudFront).
+
+Note in AWS the region we use is `us-west-2`.
+
+### Buy new domain
+
+Buy a domain for the petition in Namecheap.
+
+### Copy existing site
+
+Make a copy of an existing petition website, such as the most recently created
+one. Update the petition component props, Google Analytics tag, `.env` files,
+layout file and homepage files, images and any other references to the old
+petition.
+
+See [environment variables](#environment-variables) for details about the `.env`
+files.
+
+### Image optimization
+
+Convert any new PNGs to JPGs. Make sure JPGs are a reasonable size.
+
+### Mail petition service
+
+Update the service with configuration to send mail from the petition domain.
+
+### Captcha
+
+Add the petition domain to the captcha in Google reCAPTCHA Admin.
+
+### Create new identity in AWS SES
+
+Set up MX records.
+
+Set up and verify DKIM.
+
+Create and verify a custom MAIL FROM with domain `mail.<petition-domain>`.
+This is just for identifying the server that sends mail. No mail is actually
+sent from or received at this domain.
+
+### Create new bucket in AWS S3
+
+Allow all public access unless you want to figure out permissions for deployment
+and CloudFront.
+
+Under properties, enable static website hosting.
+Set the index document to `index.html`.
+
+### Create new distribution in CloudFront
+
+Target the S3 bucket's website endpoint. If turned on static website hosting
+in S3, AWS will prompt you to set it correctly when you update the "origin
+domain" field.
+
+Example:
+
+- Incorrect:
+  - `helpthechickens.com.s3.us-west-2.amazonaws.com`
+- Correct:
+  - `helpthechickens.com.s3-website-us-west-2.amazonaws.com`
+
+Set the alternate domain name to the petition domain (without any www).
+
+For "Custom SSL certificate" certificate, click "Request certificate", and
+create a certificate for www. and @ domains, and verify ownership with DNS
+records.
+
+Set the price class to "Use only North America and Europe" (do not use all edge
+locations).
+
+Set behavior -> viewer protocol policy: "Redirect HTTP to HTTPS".
+
+Check the box to enable IPv6 support.
+
+All other settings left as default.
+
+### Point domain to CloudFront
+
+Go to namecheap advanced dns for the domain, create CNAME record `@` pointing
+to the domain name of the CloudFront distribution found in AWS Console on the
+"General" tab for the distribution.
+
+### Create and run GitHub workflow
+
+Create a workflow file in .github/workflows and follow the other files as an
+example.
+
+Add variables and secrets in the GitHub repo settings:
+
+- https://github.com/dxe/helptheducks.com/settings/variables/actions
+- https://github.com/dxe/helptheducks.com/settings/secrets/actions
+
+Run the workflow by pushing to the main branch or running it manually from here:
+
+- https://github.com/dxe/helptheducks.com/actions
+
+### TODO
+
+- forwarding `www.` domain with Cloudflare.
```

Comment:
i think i typically just do this w/ cloudfront or s3 somehow?

Reply:
done